src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnosticsForSpan.cs

```diff
@@ -175,19 +175,13 @@ static bool ShouldIncludeAnalyzer(
                 if (!priorityProvider.MatchesPriority(analyzer))
                     return false;
 
-                // Special case DocumentDiagnosticAnalyzer to never skip these document analyzers
-                // based on 'shouldIncludeDiagnostic' predicate. More specifically, TS has special document
-                // analyzer which report 0 supported diagnostics, but we always want to execute it.
+                // Special case DocumentDiagnosticAnalyzer to never skip these document analyzers based on
+                // 'shouldIncludeDiagnostic' predicate. More specifically, TS has special document analyzer which report
```

Comment:
> TS has special document analyzer which report
                // 0 supported diagnostics, but we always want to execute it.

is this still true, or has it moved off of roslyn?

Reply:
literally in the process of moving them off :)   PR should go in this week.s